### ISSUE TITLE
The purpose

### ISSUE THREAD

**Issue by Smlangari** [devnetf/unity3d-rrt#1]
Hello, I want to run the code on Unity, but there is something wrong... Is the project complete?
Thanks
Mohammad

**Comment by devnetf** [comment]
Yes, the project is complete but I may forgot to commit certain meta files in the last push. Which version of Unity you have? I may send you a copy of the working one by email.

**Comment by Smlangari** [comment]
The Unity 4.3.4. I'll appreciate if you send the working one to my email. Actually, I am interested to know more about this project! For example, what type of RRT is this? For what purpose didi this project is accomplish? etc. My email is Smlangari@gmail.com
Thanks in advance

**Comment by devnetf** [comment]
The project is too big for email so I just updated the repo! You can try it now. 
This is the Course Project for my Robotics and Intelligent Systems Course at McGill University. I sent you a report of the RRT. You can check out my demo at https://www.youtube.com/watch?v=9jQ0F6StjE8
